CribbageTournamentTracker/src/components/Header.svelte

```diff
@@ -1,7 +1,11 @@
 <script>
 	import WebsiteLogo from '../lib/images/cribbage-logo.svg';
+	import BellIcon from '../lib/images/bell.svg';
+	import UserIcon from '../lib/images/user.svg';
+	import ExitIcon from '../lib/images/exit.svg';
```

Comment:
nit: For now, I think this is okay, but in the future we should update this to only use one import for all out icons. Can you add a comment here foj so we can remember to update this? Something perhaps like:

`TODO: update imports so we only use one`